### PR TITLE
Add css / Add footer in index.html

### PR DESCRIPTION
**Pull request by mimori**
@artem-tim 
PR with less impact on other files.
Please check it.

- Add css
- Add footer in index.html

### CODE REVIEW

**Comment by mimori** [comment]
@artem-tim 

### SignIn
SignIn steps in Video capture: https://diverta.gyazo.com/b8e2a77621c0c9eb84bd65bbf43b8ad1

 I added static components include modal header and footer.
Please implement these capture comment.

SignInStep2:
https://diverta.gyazo.com/3cacf5cbb4c52cff7aeed80aa494d52a

SignInStep3:
https://diverta.gyazo.com/2cb1e0a1102017974137f96e6830969b

File: src/components/modal_pages/SignInStep2.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<template>
+
+  <!-- ToDo: Step2 - Download -->
+  <section class="docdog-modal">
+    <header class="docdog-modal__head">
```
Comment (artem-tim) [inline src/components/modal_pages/SignInStep2.vue:5]: Header part here is shared for all Modal pages right ?

File: src/components/modal_pages/SignInStep2.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            </div>
+            <div class="docdog-card__body">
+              <p class="docdog-card__body__title">Kuroco説明資料</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Modal Footer -->
```
Comment (artem-tim) [inline src/components/modal_pages/SignInStep2.vue:40]: Footer is also shared right ?

**Review by artem-tim — COMMENTED** [review]
@mimori Sorry, I did not explain to you how Modal component is structured

Basically, all components under `src/components/modal_pages` are Modal Body
Modal Header and Modal Footer are shared here : https://github.com/diverta/docdog-lib/blob/master/src/components/Modal.vue#L4-L26

Components in `src/components/modal_pages` are supposed to be inserted into the default `<slot>` of the Modal : https://github.com/diverta/docdog-lib/blob/master/src/components/Modal.vue#L19

Which page to display is chosen by the PageController : https://github.com/diverta/docdog-lib/blob/master/src/App.vue#L25-L30

I will add new page logic myself, but you don't need to copy Headers and Footers for every `src/components/modal_pages` - they only need to contain Modal Body (`docdog-modal__body__section` div)

**Comment by artem-tim** [comment]
Thank you for the static design.
I will merge this and add pages myself, but I wanted to let you know about global Modal

File: src/components/modal_pages/SignInStep2.vue (hunk 1 above)
Comment (mimori) [reply src/components/modal_pages/SignInStep2.vue:5]: Yes, it isn't need a new logic.

File: src/components/modal_pages/SignInStep2.vue (hunk 2 above)
Comment (mimori) [reply src/components/modal_pages/SignInStep2.vue:40]: Footer may or may not have elements depending on the modal.

I want to change include modal footer.
https://diverta.gyazo.com/8412253ea9ecc19b1e79bb96e3ea0962

It seemed that I could only write html for modal-body,
so I coded all modal code for now.

Comment (artem-tim) [reply src/components/modal_pages/SignInStep2.vue:40]: I see. I will change to allow custom header or footer per page

**Pull request merged**